doc/integrator/upgrade_application.rst.mako

```diff
@@ -37,19 +37,19 @@ Pull the latest version of the Docker image:
 
 Where ``<version>`` is ``${major_version}.0`` for the first stable release of the version ``${major_version}``.
 
-For Docker:
+And for non-Docker:
 
 .. prompt:: bash
 
-   ./docker-run --home --image=camptocamp/geomapfish-build --version=<version> \
-       c2cupgrade --force-docker --new-makefile=Makefile --makefile=<package>.mk
+   ./docker-run --home --image=camptocamp/geomapfish-build \
+       c2cupgrade --nondocker --makefile=testgeomapfish.mk
```

Comment:
`<package>.mk` instead of `testgeomapfish.mk` ?

Reply:
Many thanks, fixed :-)